src/game_api/rpc.cpp

```diff
@@ -1124,7 +1124,7 @@ void extinguish_particles(ParticleEmitterInfo* particle_emitter)
     if (particle_emitter != nullptr)
     {
         generic_free(particle_emitter->emitted_particles.memory);

```

Comment:
Moving the particle count into the struct means this is now pointing to a different point in memory, so I'm once again not sure if this will still work or if it needs to be updated to free at the right spot.

Reply:
The memory pointer still points to the same address, nothing has changed, you're good.